nautobot/extras/models/roles.py

```diff
@@ -47,13 +47,13 @@ class RoleModelMixin(models.Model):
     Abstract base class for any model which may have roles.
     """
 
-    role = RoleField()
+    role = RoleField(null=True, blank=True)
 
     class Meta:
         abstract = True
 
 
-class RoleRequiredRoleModelMixin(RoleModelMixin):
+class RoleRequiredRoleModelMixin(models.Model):
```

Comment:
Why was this problematic, for my education?

Reply:
Subclassing an abstract model to create another abstract model that is overloading the same field is apparently not doing what it was intended to do according to the bug report. I was able to confirm the somewhat confusing behavior.